third_party/nvfuser/csrc/contiguity.h

```diff
@@ -55,6 +55,8 @@ class OrderedIdInformation : public OptInDispatch {
 
   void handle(Swizzle2D* swizzle) override;
 
+  void handle(Resize* expand) override;
```

Comment:
Should we use names more consistently? Currently, we are using resize/expand. Should we replace all "expand" with resize?

Reply:
I started with `IterDomain::expand` initially as I was only thinking about padding, but realized it's not a good name when adding support of slicing, so renamed it to `resize`. Looks like there were still some using the old name. Renamed this. I don't think there's no such name anymore.